packages/components/src/components/date-picker/date-picker-native.jsx

```diff
@@ -85,7 +85,17 @@ const Native = ({
                 onFocus={handleFocus}
                 disabled={disabled}
                 onChange={e => {
-                    onSelect(e.target.value);
+                    let newValue = e.target.value;
```

Comment:
can we use snake_case for `newValue`?
Refer: Our js guideline - [Link](https://github.com/binary-com/deriv-app/blob/master/docs/JavaScript/README.md)

Reply:
Yes sure I will update it thanks